### PR TITLE
Draft vignette porting The Bank: Part II from SimPy

### PR DESCRIPTION
**Pull request by nacnudus**
I wasn't sure what to name the file, so it is `simmer-08-bank-2.Rmd` for now.

### CODE REVIEW

**Comment by Enchufa2** [comment]
Many thanks! I'm travelling to the useR!2017 conference today, so I'll review this PR in another moment.

For now, please rename the file as `simmer-04-bank-2.Rmd`.

**Comment by nacnudus** [comment]
No hurry, enjoy useR!  Best wishes for the Simmer presentation.

**Comment by Enchufa2** [comment]
The Travis build has errored because there are some namespace conflicts with `dplyr`. To avoid this, as you can see in the other vignettes, I prefer not to load `dplyr`, but to call its functions explicitly from its namespace, i.e., `dplyr::select()` instead of `select()`. So please remove `library(dplyr)` and prepend `dplyr::` to every `dplyr` method.

**Comment by nacnudus** [comment]
Thanks for the tip, I was struggling to figure that one out.

**Comment by Enchufa2** [comment]
Some more comments:

- There is a typo: "You man all enter".
- In "Monitors", I would specify that the `get_mon_*()` functions return a data frame (that is growing during the simulation, so that, although possible, it is computationally expensive to call them from inside a trajectory), while the others return a numeric.
- Please add `simmer.plot` to the list of requirements in the first chunk.
- In "Plotting a Histogram of Monitor results", you talk about line numbers, but I can't quite understand where you are pointing to (maybe I read it too quickly?)
- Please update the NEWS file with the following under the "Minor changes and fixes" section:

> New "The Bank Tutorial: Part II" vignette, by Duncan Garmonsway @nacnudus (#106).

File: vignettes/simmer-04-bank-2.Rmd [hunk 1]
```diff
@@ -406,7 +406,7 @@ door_schedule <- schedule(c(0, openTime), c(0, Inf))
 doorman <-
   trajectory() %>%
   timeout(openTime) %>%
-  log_("Ladies and Gentlemen! You man all enter.")
+  log_("Ladies and Gentlemen! You can all enter.")
```
Comment (Enchufa2) [inline vignettes/simmer-04-bank-2.Rmd]: I thought that was a "may". 😄

**Comment by Enchufa2** [comment]
Many thanks, merged!

File: vignettes/simmer-04-bank-2.Rmd (hunk 1 above)
Comment (nacnudus) [reply vignettes/simmer-04-bank-2.Rmd]: Haha, you're right!  Too late now ;-)
temp

**Pull request merged**